.changeset/nine-crews-flow.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/access-control': patch
+'@keystonejs/keystone': patch
```

Comment:
```suggestion
'@keystonejs/access-control': major
'@keystonejs/keystone': major
```

The API for calling `validateFieldAccessControl` has changed.

Reply:
I think we should also update the message to give a little more detail of what's changed. In addition to `validateFieldAccessControl`, `checkListAccess` and `checkFieldAccess` also return promises now? We need to flag and change to external APIs here.

Reply:
Some others that changed: `validateCustomAccessControl`, `validateListAccessControl`, `validateAuthAccessControl `, `checkAccess` also some of what is returned by `getGraphQlContext`.

Basically any public method that is now async needs to be mentioned because people could potentially be using it.